app/coverage.gradle

```diff
@@ -5,9 +5,9 @@ jacoco {
 }
 
 // change the buildType and productFlavor fields here if the project uses different build variant for testing
-def buildType = "debug";
-def productFlavor = "";
-def buildVariant = "${productFlavor.capitalize()}${buildType.capitalize()}";
+def buildType = "debug"
+def productFlavor = ""
+def buildVariant = "${productFlavor.capitalize()}${buildType.capitalize()}"
```

Comment:
These also get corrected in #21 but that should be pretty easy to resolve conflicts on